src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -60,49 +60,7 @@
         </a>.
 
       </remarks>
-      <example>
```

Comment:
>example [](start = 7, length = 7)

Created a separate issue #1715 to (1) add mlcontext extension for TFTransform (2) add TFTransform example to Microsoft.ML.Samples